app/javascript/stylesheets/components/_flashes.scss

```diff
@@ -8,14 +8,6 @@
     @include flash;
     color: govuk-colour("green");
   }
-  &.success {
-    @include flash;
-    color: govuk-colour("green");
-  }
-  &.error {
-    @include flash;
-    color: $govuk-error-colour;
-  }
```

Comment:
`.success` css is identical to c for `.notice`, and `.error` css is identical to css for `.alert`.